cypress/integration/10-contributionFlow.createProfile.test.js

```diff
@@ -30,11 +25,6 @@ describe('Contribution Flow: Create profile', () => {
   it('Organization profile', () => {
     cy.visit('/apex/donate');
 
-    // Go to CreateProfile
-    cy.get('#content button')
-      .contains('Join Free')
-      .click();
-
```

Comment:
If we rolled back the "Sign In" / "Create Account" changes, can we rollback the change to tests?

Reply:
No because this is for the /donate page where the default form should be create an account.